__docs__/wonder-blocks-link/link.argtypes.ts

```diff
@@ -9,6 +10,17 @@ export default {
         type: {name: "string", required: true},
     },
 
+    endIcon: {
+        control: {type: "select"},
+        description: `Icon to appear after the link label. If
+        \`target="_blank"\` and an \`endIcon\` is passed in, \`endIcon\` will
+        override the default \`externalIcon\`.`,
+        options: icons,
```

Comment:
Can you add a `none: null` option to the `startIcon` and `endIcon` options? Currently, there's no way to get rid of the icon once you select one. I believe you can do this by using:

```
options: {none: null, ...icons},
```